src/picogl.js

```diff
@@ -94,3 +108,11 @@ export const PicoGL = Object.assign({
         return new App(gl);
     }
 }, GL);
+
+/**
+ * This is a hack to be able to document the default export :(
+ * @type {PicoGL}
+ * @name exportDefaultPicoGL
```

Comment:
Would it be worth trying this jsdoc plugin? https://www.npmjs.com/package/jsdoc-export-default-interop

It's mentioned in the tsd-jsdoc docs as a solution: https://github.com/englercj/tsd-jsdoc#default-exports

Reply:
I've tried this but it didn't help.

The problem here is more related to this issue:
https://github.com/englercj/tsd-jsdoc/issues/64

Basically, because of JSDoc limitations `tsd-jsdoc` generates ambient type declarations that are not compatible with ES6 module style exports, to circumvent this issue I run a replace script that simply replaces all `declare` keywords for `export`. This works fine for all named exports but doesn't work at all for default exports because in typescript there is no notion of a `declare default`, so this is the best option I could come up with given the limitations in the toolchain. It's a hack, but hopefully can be removed in the future once JSDoc and tsd-jsdoc solve the issues mentioned above.